src/util.h

```diff
@@ -434,23 +434,23 @@ void RenameThreadPool(ctpl::thread_pool& tp, const char* baseName);
 /**
  * .. and a wrapper that just calls func once
  */
-template <typename Callable> void TraceThread(const char* name,  Callable func)
+template <typename Callable> void TraceThread(std::string name,  Callable func)
 {
-    std::string s = strprintf("dash-%s", name);
+    std::string s = "dash-" + name;
     RenameThread(s.c_str());
     try
     {
-        LogPrintf("%s thread start\n", name);
+        LogPrintf("%s thread start\n", name.c_str());
```

Comment:
No need to, `LogPrintf` works with `std::string` just fine. Should be reverted for all LogPrintf-s.

Reply:
Makes sense :)